tests/complex_proxy/test_complex_vector.py

```diff
@@ -5,6 +5,13 @@
 import pytest
 
 
+def assemble(form, tensor=None):
+    if tensor:
```

Comment:
I don't think this if-statement is necessary. Can't you just pass `None` through if `tensor` isn't given?